README.md

```diff
@@ -73,6 +73,7 @@ Secondly, try pressing Ctrl+t. The tabs are hidden when you only have one tab by
 ## Prerequisites
 
 * You must be running Windows 1903 (build >= 10.0.18362.0) or above in order to run Windows Terminal
+  - You will need to register and setup Windows Insider Program on your device to get access to the latest builds.

```

Comment:
This has a disclaimer/time-sensitivity. It will be fine with the 1903 RTM once it is available broadly and won't need an Insider registration from there forward. Can you indicate that somehow?

I'm happy to have the note for the short-term, but I want it to expire once the RTM is broadly available in case folks don't want to stay on the Insider train for whatever reason.

Reply:
Sure thing, I have updated the readme to be more descriptive as well as add a time-sensitive note.